Dockerfile

```diff
@@ -24,7 +25,12 @@ RUN go build\
 
 FROM registry.access.redhat.com/ubi8/ubi-minimal AS final
 RUN microdnf install --disablerepo=* --enablerepo=ubi-8-baseos --enablerepo=ubi-8-appstream tar
-RUN curl -L -o /usr/local/bin/dumb-init https://github.com/Yelp/dumb-init/releases/download/v1.2.2/dumb-init_1.2.2_amd64 && chmod +x /usr/local/bin/dumb-init 
+RUN case "$(uname -m)" in \
```

Comment:
this is a little bit of a shame, but can't think of a more graceful way

Reply:
Yeah, I don't know when the syntax/backend updates that allow the "buildx" extra variables will be usable.

Reply:
...or rhel pick up / yelp publish rpms and microdnf would take care of the arch parsing

Reply:
Yeah, I don't recall if there's an intent to package `dumb-init,` and I certainly don't want to be on the hook for an `rpm`.

The other options are:

1. write a little bit of code to catch `SIGCHLD` and ditch `dumb-init`
2. remove the exec calls that necessitate pid 1 handling (currently just `rpm`, I think)